lib/ui/recommendation-row.js

```diff
@@ -256,8 +256,10 @@ RecommendationRow.prototype = {
     const val = key.split('.').reduce(function(o, x) {
       return (typeof o == 'undefined' || o === null) ? o : o[x];
     }, this._data);
-    const isEmptyObject = (typeof val == 'object' && !Object.keys(val).length);
-    return (typeof val == 'undefined' || !val || isEmptyObject) ? null : val;
+    let isEmptyObject = val => {
+      return val && typeof val == 'object' && !Object.keys(val).length;
```

Comment:
you can skip the truthy check here, since you do it on line 262 before this function is called: `return typeof val == 'object' && !Object.keys(val).length`